tests/common/features/datetimeutc.rs

```diff
@@ -0,0 +1,16 @@
+use sea_orm::entity::prelude::*;
+
+#[derive(Clone, Debug, PartialEq, DeriveEntityModel)]
+#[sea_orm(table_name = "satellites")]
+pub struct Model {
+    #[sea_orm(primary_key)]
+    pub id: i32,
+    pub satellite_name: String,
+    pub launch_date: DateTimeUtc,
+    pub deployment_date: DateTimeUtc,
+}
```

Comment:
Hey @charleschege, you could set one of the timestamp column to nullable. This will solve the problem, let say changing the type of `deployment_date` attribute to `Option<DateTimeUtc>`

Reply:
Thanks @billy1624

Reply:
I think I am a little bit confused about some results, does using the derive macro yield different results than the `Table::create` statement?

Reply:
For example, on run https://github.com/charleschege/sea-orm/runs/4770827851?check_suite_focus=true

using https://github.com/SeaQL/sea-orm/pull/429/files/1227ad8252815c72b19c4a6aa27e998d3f1e5330#diff-db71d1612ed300a9a9a38e99d0096890446cc04f150507455638b35110d81bf0 results in that error

Reply:
But using the manual method
```
sea_query::Table::create()
                .table(Entity)
                .col(
                    ColumnDef::new(Column::Id)
                        .integer()
                        .not_null()
                        .auto_increment()
                        .primary_key(),
                )
                .col(ColumnDef::new(Column::SatelliteName).string().not_null())
                .col(
                    ColumnDef::new(Column::LaunchDate)
                        .timestamp_with_time_zone()
                        .not_null(),
                )
                .col(
                    ColumnDef::new(Column::DeploymentDate)
                        .timestamp_with_time_zone()
                        .not_null(),
                )
                .if_not_exists()
                .to_owned();
```

Assertion doesn't fail, @billy1624 is there a way to ensure fields are not null using the derive macro ? The module docs in https://docs.rs/sea-orm/latest/sea_orm/ dont show how to do this if possible

Reply:
If `launch_date` is nullable then the table create statement will be like...

```rust
.col(
    ColumnDef::new(Column:: LaunchDate)
        .timestamp_with_time_zone(),
)
```